packages/psss-server/src/routes/reports/SubmitConfirmedCountryWeeklyReportRoute.ts

```diff
@@ -0,0 +1,90 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import { RespondingError } from '@tupaia/utils';
+import { Route } from '../Route';
+import { PSSS_WEEKLY_REPORT_CODE } from '../../constants';
+
+const SURVEY_CODE = 'PSSS_Confirmed_WNR';
+
+type ConfirmedWeeklyReportAnswers = {
+  PSSS_Confirmed_Sites: number;
+  PSSS_Confirmed_Sites_Reported: number;
+  PSSS_Confirmed_AFR_Cases: number;
+  PSSS_Confirmed_DIA_Cases: number;
+  PSSS_Confirmed_ILI_Cases: number;
+  PSSS_Confirmed_PF_Cases: number;
+  PSSS_Confirmed_DLI_Cases: number;
+};
```

Comment:
I'm quite a fan of defining in-file types like this. It helps me do type checking easier and generally makes things clearer to me. However I have a feeling you're meant to define all types in the root `/types` file? Any thoughts on this?

Reply:
I've had the same question in my mind but don't know the answer!

Reply:
I think generally we should try and sketch up some guidelines for good typescript practices. I'll need to do some more research and develop more experience. If you're happy with this I'd like to keep it?

Reply:
Happy as larry